### PR TITLE
Update licenses mentioned in issue #983

### PR DESCRIPTION
**Pull request by kirkins**
https://github.com/Kickball/awesome-selfhosted/issues/983

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines. 
        
    ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language```
- [x] Your additions are ordered alphabetically.
- [x] Your additions are [free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md).
- [x] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant issues or PRs.
- [x] Any category you are creating has the minimum requirement of 3 items.

### CODE REVIEW

**Comment by danger-awesome-Selfhosted** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please check PR guidelines and check the boxes.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Found 1 link issue</td>
    </tr>
  </tbody>
</table>

#### Link issue by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
3 | [404](https://httpstatuses.com/404) | http://boards.tomcanac.com/ 


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by kirkins** [comment]
Updated demo link after the danger-awesome-Selfhosted bought picked up a broken one.

File: README.md [hunk 1]
```diff
@@ -906,7 +906,7 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
...
   * [MediaDrop](http://mediadrop.net/) - A video, audio and podcast publication platform. ([Source Code](https://github.com/mediadrop/mediadrop)) `GPLv3` `Python`
   * [Mediagoblin](http://mediagoblin.org) - A Free software media publishing platform that anyone can run. You can think of it as a decentralized alternative to Flickr, YouTube, SoundCloud, etc. `AGPLv3` `Python`
-  * [MinigalNano](https://github.com/sebsauvage/MinigalNano) - MinigalNano is a very simple image gallery. It adheres to the KISS principle and is very easy to install. MinigalNano does not have a web admin interface: You just upload your images in the photo folder on your server (using FTP, SFTP). ([Demo](http://boards.tomcanac.com/)) `GPLv3/CCBYNCNDv4` `PHP`
+  * [MinigalNano](https://github.com/sebsauvage/MinigalNano) - MinigalNano is a very simple image gallery. It adheres to the KISS principle and is very easy to install. MinigalNano does not have a web admin interface: You just upload your images in the photo folder on your server (using FTP, SFTP). ([Demo](www.minigal.dk/minigalnano/)) `AGPLv3` `PHP`
```
Comment (nodiscc) [inline README.md]: www.minigal.dk/minigalnano is not the same project, sebsauvage/MinigalNano is based on it but the dk version is ancient; it is on 0.3.5, 0.3.7 was released [23 mar 2013](https://github.com/sebsauvage/MinigalNano/commit/c6b911cc2947245479b475d96dccd091902ab5c3), so at least 4 years old.

I was not aware the minigalnano demo was down/removed from the README :/ I think you can simply remove the link.

Comment (kirkins) [inline README.md]: ah, I just deleted the link

**Pull request merged**